decidim-admin/app/queries/decidim/admin/newsletter_recipients.rb

```diff
@@ -76,7 +76,13 @@ def participant_ids
         participant_ids = []
         spaces.each do |space|
           available_components = Decidim.component_manifests.map { |m| m.name.to_s if m.newsletter_participant_entities.present? }.compact
-          Decidim::Component.where(id: space.component_ids, manifest_name: available_components).published.each do |component|
+          component_ids = if space.respond_to? :question_ids
```

Comment:
Have you considered aliasing `question_ids` to `component_ids` on Consultations? This way we'd keep the same interface for all spaces.